src/pages/guide/rtl/en/index.md

```diff
@@ -12,22 +12,27 @@ Make sure the `dir` attribute is set on the body:
 <body dir="rtl"></body>
 ```
 
-## 2.IntlProvider
+### 2.IntlProvider
 
-Set the rtl attribute on the IntlProvider component to configure all components to support RTL.
+Set the `rtl` props on the IntlProvider component to configure all components to support RTL.
 
 ```jsx
-ReactDOM.render(
-  <IntlProvider rtl>
-    <App />
-  </IntlProvider>,
-  document.getElementById('root')
-);
+function RTL(props) {
+  return <IntlProvider rtl>{props.children}</IntlProvider>;
+}
 ```
 
-## 3.postcss-rtl
+### 3.Post-Processing CSS
```

Comment:
@simonguo We already switched to generating full RTL version, The use of bidi version will have some discrepancies and wont work well out of the box. I recommend keeping the officially supported method only.